include/mbgl/ios/MGLAnnotationImage.h

```diff
@@ -13,12 +13,12 @@ NS_ASSUME_NONNULL_BEGIN
 *   @param image The image to be displayed for the annotation.
 *   @param reuseIdentifier The string that identifies that this annotation image is reusable.
 *   @return The initialized annotation image object or `nil` if there was a problem initializing the object. */
-+ (instancetype)annotationImageWithImage:(UIImage *)image reuseIdentifier:(NSString *)reuseIdentifier;
++ (instancetype)annotationImageWithImage:(UIImage *)image reuseIdentifier:(nullable NSString *)reuseIdentifier;
```

Comment:
Hi,
That was a reminder to make it nullable as in the official MapKit. This identifier is used to recycle the instance for performance reason, it should not be used internally like today to reference the image view in a dictionary or even deeper with the sprites